### PR TITLE
Changes for release v0.8.0

### PR DESCRIPTION
**Pull request by dreamtalen**

### CODE REVIEW

**Comment by antoninbas** [comment]
Your `release-0.8.0` branch is not correct. It should be the same as `main` right now?

**Comment by dreamtalen** [comment]
> Your `release-0.8.0` branch is not correct. It should be the same as `main` right now?

I got your point, it seems that our release workflow differs from that of Antrea. 
I created the `release-0.8.0` branch based on `release-0.7.0` and cherry-picked commits for this release.
Some example PR of previous releases:
https://github.com/vmware/go-ipfix/pull/299
https://github.com/vmware/go-ipfix/pull/309

**Comment by antoninbas** [comment]
@dreamtalen ok, I'll let @heanlan review the PR then :)

**Comment by heanlan** [comment]
@dreamtalen could you check the difference between your branch and the main branch by running `git diff`? `release-0.7.0` was created based off the main branch at that time. So I suppose there should be no differences for your branch and the main 🤔

**Comment by dreamtalen** [comment]
> @dreamtalen could you check the difference between your branch and the main branch by running `git diff`? `release-0.7.0` was created based off the main branch at that time. So I suppose there should be no differences for your branch and the main 🤔

Yes, there are no differences except `CHANGELOG.md` and `VERSION` file

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.  The format
...
+- Add L7 visibility fields. (#315, @tushartathgur)
+### Changed
+- Change datatype of flowEndSecondsFromSourceNode and
+  flowEndSecondsFromDestinationNode. (#320, @tushartathgur)
+- Improve handling of string IEs. (#322, @antoninbas)
+- Avoid error logs in BenchmarkMultipleExportersToCollector. (#323,
+  @antoninbas)
+- Improve integration tests. (#325, @antoninbas)
+### Fixed
+- Fix aggregation bug for throughput common fields. (#324, @antoninbas)
```
Comment (heanlan) [inline CHANGELOG.md:18]: "Fix aggregation bug that throughput common fields should be only updated when receiving record is the latest"

**Review by heanlan — APPROVED** [review]
LGTM

**Pull request merged**